Friendly-plans/app/src/main/res/values/strings.xml

```diff
@@ -71,8 +70,15 @@
   <string name="step_saved_message">Step saved</string>
   <string name="steps_reordered_message">Steps reordered and saved</string>
   <string name ="search">Search</string>
-  <string name="create_plan_tasks_list_info">Step 2.1: Add new tasks and put them in order</string>
-  <string name="create_plan_add_tasks">Add tasks</string>
+  <string name="create_plan_tasks_list_info_type_1">Step 2.1: Add new tasks and put them in order</string>
+  <string name="create_plan_tasks_list_info_type_2">Step 2.1: Add new prizes and put them in order</string>
+  <string name="create_plan_tasks_list_info_type_3">Step 2.1: Add new interactions and put them in order</string>
+  <string name="create_plan_add_tasks_type_1">Add tasks</string>
```

Comment:
This screen is used for adding and displaying a list of tasks, prizes, interactions assigned. I would rename the buttons to Tasks list, Prizes list, Interaction list, because of assigning is performed in the steps.